### DISCUSSION TITLE
fps drops and latency problems

### DISCUSSION THREAD

**Discussion by ytGrapeFrootZ349** [ppy/osu#23337] [Q&A] [answered]
Whenever I run lazer this error shows up and I get terrible fps drops and latency problems
"Running osu! as administrator not improve performance, may break integrations and poses a security risk. Please run the game as a normal user"
I checked that I am not running lazer as administrator and I also don't have compatibility mode enabled
I also don't have this problem in stable
plz help

**Comment by Theighlin** [comment]
Have you modified your windows install?

**Reply by ytGrapeFrootZ349** [reply]
I don't think so, but I have done tweaks on my system

**Comment by ytGrapeFrootZ349** [comment]
I fixed the issue after doing resting and reinstalling windows
might have had something to do with the tweaks i did before

**Comment by ytGrapeFrootZ349** [comment, marked as answer]
apparently it was user account control and spectre & meltdown protection
i disabled those and it led to osu being all wonky and stuff
just re-enable uac and meltdown protection and it should fix itself
